README.md

```diff
@@ -3,6 +3,8 @@
 # Module 3: Calculator
 
 > i made up Calculator with reactjs 
+> i used state
+> i used setState
```

Comment:
- [ ] Please deploy the project online and include the live deploy link in the readme.